src/agent/firebase-controller.ts

```diff
@@ -300,6 +320,26 @@ export class FirebaseController implements Controller {
         callback(e, []);
       }
     );
+
+    this.startMarkingDebuggeeActive();
+  }
+
+  startMarkingDebuggeeActive() {
+    console.log(`starting to mark every ${this.markActivePeriod} ms`);
+    this.markActiveInterval = setInterval(() => {
```

Comment:
Rename to `markActiveIntervalId` ? it wasn't until I read the setInterval documentation that I realized it returns an ID.

Reply:
Ah, interesting, and fair enough.